internal/app/client/conformance_client.go

```diff
@@ -160,6 +167,55 @@ func (i *invoker) serverStream(
 	}, nil
 }
 
+func (i *invoker) clientStream(
+	ctx context.Context,
+	req *v1alpha1.ClientCompatRequest,
+) (*v1alpha1.ClientResponseResult, error) {
+	stream := i.client.ClientStream(ctx)
+
+	// Add the specified request headers to the request
+	app.AddHeaders(req.RequestHeaders, stream.RequestHeader())
+
+	for _, msg := range req.RequestMessages {
+		csr := &v1alpha1.ClientStreamRequest{}
+		if err := msg.UnmarshalTo(csr); err != nil {
+			return nil, err
+		}
+		if err := stream.Send(csr); err != nil && errors.Is(err, io.EOF) {
```

Comment:
I think this should sleep for the delay indicated in `req` before sending, right?

Reply:
Ah yeah, good call. I actually updated the proto for `ClientCompatRequest` also. I previously had a TODO that the delay field there could be removed, but now I realize that should specify the delay for _requests_.